commune_app/tests/test_commune.py

```diff
@@ -38,3 +32,11 @@ def test_create_delete_commune(self):
         new_commune.delete()
         with pytest.raises(Exception):
             Commune.objects.get(id=new_commune.id)
+
+    def test_wallet(self, commune0):
+        commune0.wallet_charge(-1000)
+        assert commune0.wallet == 1000
+        commune0.wallet_charge(800)
+        assert commune0.wallet == 200
```

Comment:
I think it worth splitting this test into 2 different tests 
this one with test_positive_wallet_charge